### DISCUSSION TITLE
POKEDEX terminado(DEBES VERLO)

### DISCUSSION THREAD

**Discussion by enriqueorea** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#483] [Show and tell]
Hola compañeros quiero compartirles un poco de mi trabajo con el pokedex
creo que falta pulir ciertos aspectos que con el paso del tiempo y mayor conocimiento de las tecnologías podría mejorarlo
escucho sus comentarios y observaciones, les agradezco mucho

https://lovely-fox-08836e.netlify.app/

ese es el link

**Comment by khisus19** [comment]
Esta muy bonita. ¿De donde obtienes los sprites? Porque se ven muy bien

**Reply by enriqueorea** [reply]
Muchas gracias utilizo sprites.front_default para la sección de todo el pokedex y sprites.other.dream_world.front_default para el sidebar

**Reply by JLuisPrz** [reply]
El único problema de usar sprites.other.dream_world.front_default es que a partir del Pokémon 650 en adelante no se ven las imágenes.

**Reply by enriqueorea** [reply]
Tienes razon, muchas gracias lo voy a  checar

**Comment by JuanCarlosSalgado** [comment]
Muy buenas tardes me podrías decir como le hiciste para los tipos? tengo una idea de como hacerlo pero al parecer hay un error en el código

**Reply by enriqueorea** [reply]
Hola buenas tardes, los tipos vienen contenidos en un Array por lo cual para mostrarlos debes de utilizar alguna funcion que recorra el array de tipos

**Comment by HeribertoBonilla** [comment]
Wow, Me encanto! se ve sorprendente

**Comment by JuanSalvador73** [comment]
Te quedó increíble. Haznos un tuto porfa :x
